### PR TITLE
Update vscode-setup.rst (IDFGH-6810)

### PR DESCRIPTION
**Pull request by andypiper**
Improved formatting and grammar, fixed links so that they render correctly

### CODE REVIEW

File: docs/en/get-started/vscode-setup.rst [hunk 1]
```diff
@@ -4,33 +4,33 @@ Getting Started with VS Code IDE
...
+* **IDF Size Analysis Overview** presents a UI for binary size analysis.
+* `Rainmaker Cloud <https://rainmaker.espressif.com/>`_, we have inbuilt Rainmaker Cloud support where you can edit/read the state of your connected IoT devices easily.
+* **Code Coverage**, we have inbuilt code coverage support which has color highlights showing which line have been covered. We also render the existing HTML report directly inside the IDE.
 
 Bugs & Feature Requests
 =======================
 
-If you face an issue with certain feature of VS Code or VS Code in general we recommend to ask your question in the `forum <http://esp32.com/viewforum.php?f=40>`_, or open a `github issue <https://github.com/espressif/vscode-esp-idf-extension/issues/new/choose>`_ for our dev teams to review.
+If you face an issue with certain feature of VS Code or VS Code in general we recommend you ask your question in the `forum <http://esp32.com/viewforum.php?f=40>`_, or open a `GithHb Issue <https://github.com/espressif/vscode-esp-idf-extension/issues/new/choose>`_ for our dev teams to review.
```
Comment (krzychb) [inline docs/en/get-started/vscode-setup.rst:34]:
```suggestion
If you face an issue with certain feature of VS Code or VS Code in general we recommend you ask your question in the `forum <http://esp32.com/viewforum.php?f=40>`_, or open a `GitHub Issue <https://github.com/espressif/vscode-esp-idf-extension/issues/new/choose>`_ for our dev teams to review.
```

**Review by krzychb — CHANGES_REQUESTED** [review]
Hi @andypiper,
I left one small comment. The updates are great. 
Thank you for your contribution!

File: docs/en/get-started/vscode-setup.rst [hunk 2]
```diff
@@ -4,33 +4,33 @@ Getting Started with VS Code IDE
...
+* **Build**, with one-click build and multi-target build, you can easily build and deploy your applications.
+* **Flash**, with both UART and JTAG flash out-of-the-box.
+* **Monitoring**, comes with a built-in terminal, you can trigger IDF Monitor Commands from within VS Code as you are used to in traditional terminals.
+* _`Debugging <https://github.com/espressif/vscode-esp-idf-extension/blob/master/docs/tutorial/debugging.md>`_, with out-of-the-box hardware debugging and also support for postmortem debugging like coredumps, you can analyze the bugs with convenience.
+* **GUI Menu Config**, provides a simplified UI for configuring your chip.
+* **App & Heap Tracing**, provides support for collecting traces from your application, and a simplified UI for analyzing them.
+* **System View Tracing Viewer**, aims to read and display the *.svdat* files into the trace UI (we also support multiple core tracing views).
+* **IDF Size Analysis Overview** presents a UI for binary size analysis.
+* `Rainmaker Cloud <https://rainmaker.espressif.com/>`_, we have inbuilt Rainmaker Cloud support where you can edit/read the state of your connected IoT devices easily.
```
Comment (pedrominatel) [inline docs/en/get-started/vscode-setup.rst:28]: and

```
* **Rainmaker Cloud**, we have inbuilt Rainmaker Cloud support where you can edit/read the state of your connected IoT devices easily. For more information see this `page <https://rainmaker.espressif.com/>`_
``

**Review by pedrominatel — COMMENTED** [review]
Please take a look @andypiper.

**Comment by pedrominatel** [comment]
Hi @andypiper, thanks for the valuable contribution!

I see some issues on building this change using our docs-build. Please see my suggestions.

Thanks!

**Comment by andypiper** [comment]
Thanks for the review and apologies for those errors, I like the suggestions as better ways to get uniform formatting in the list here. I've rechecked the links this time.

**Comment by pedrominatel** [comment]
LGTM! Thanks @andypiper

**Comment by krzychb** [comment]
I have taken this PR into the internal queue for merging.
Thank you @andypiper!

**Comment by krzychb** [comment]
Merged in https://github.com/espressif/esp-idf/commit/6249b3d6718372c12ae96dd288181cd058abeca4

**Pull request closed without merging**